scene/animation/root_motion_view.cpp

```diff
@@ -134,9 +133,9 @@ void RootMotionView::_notification(int p_what) {
 					Vector3 from(i * cell_size, 0, j * cell_size);
 					Vector3 from_i((i + 1) * cell_size, 0, j * cell_size);
 					Vector3 from_j(i * cell_size, 0, (j + 1) * cell_size);
-					from = accumulated.xform(from);
-					from_i = accumulated.xform(from_i);
-					from_j = accumulated.xform(from_j);
+					from = accumulated.xform_inv(from);
```

Comment:
Is this ok? remember that if scale is present, xform_inv will not work.

Reply:
@reduz transform's scale has been orthonormalized in line 116, so always accumulated's scale is should be 1.

Reply:
alright looks good then